tests/test_simulation.py

```diff
@@ -25,53 +25,53 @@
 from pulser.waveforms import BlackmanWaveform, ConstantWaveform, RampWaveform
 from pulser_simulation import SimConfig, Simulation
 
-q_dict = {
-    "control1": np.array([-4.0, 0.0]),
-    "target": np.array([0.0, 4.0]),
-    "control2": np.array([4.0, 0.0]),
-}
-reg = Register(q_dict)
-
-duration = 1000
-pi = Pulse.ConstantDetuning(BlackmanWaveform(duration, np.pi), 0.0, 0)
-twopi = Pulse.ConstantDetuning(BlackmanWaveform(duration, 2 * np.pi), 0.0, 0)
-pi_Y = Pulse.ConstantDetuning(
-    BlackmanWaveform(duration, np.pi), 0.0, -np.pi / 2
-)
-
-seq = Sequence(reg, Chadoq2)
-
-# Declare Channels
-seq.declare_channel("ryd", "rydberg_local", "control1")
-seq.declare_channel("raman", "raman_local", "control1")
-
-d = 0  # Pulse Duration
-
-# Prepare state 'hhh':
-seq.add(pi_Y, "raman")
-seq.target("target", "raman")
-seq.add(pi_Y, "raman")
-seq.target("control2", "raman")
-seq.add(pi_Y, "raman")
-d += 3
-
-prep_state = qutip.tensor([qutip.basis(3, 2) for _ in range(3)])
-
-# Write CCZ sequence:
-seq.add(pi, "ryd", protocol="wait-for-all")
-seq.target("control2", "ryd")
-seq.add(pi, "ryd")
-seq.target("target", "ryd")
-seq.add(twopi, "ryd")
-seq.target("control2", "ryd")
-seq.add(pi, "ryd")
-seq.target("control1", "ryd")
-seq.add(pi, "ryd")
-d += 5
-
-# Add a ConstantWaveform part to testout the drawing procedure
-seq.add(Pulse.ConstantPulse(duration, 1, 0, 0), "ryd")
-d += 1
+
```

Comment:
Why not putting all the fixture in conftest.py?

Reply:
Because I don't want these to be shared with other test files, especially when they're named `seq` and `reg`.

Reply:
Make sense